### ISSUE TITLE
[REQUEST] Clickable paper checklist

### ISSUE THREAD

**Issue by St54Kevin** [flybywiresim/a32nx#1270]
**Is your feature request related to a problem? Please describe.**

IRL we use laminated paper checklists stowed in the checklist holder located on each side of the pedestal. Would be ncie to have some modeled there and able to popup with a click for fast reference. Be aware that I mean the real checklist not the procedures which are often misunderstood as a checklist (like the one in MSFS). 

**Describe the feature you'd like to see implemented**

**References**

I have to get a better picture of one of ours but here is an bad quality example taken from Google search. Current Airbus checklist is way shorter. 
![Monarch-Airlines-Normal-Checklist-A320-A321-Laminated](https://user-images.githubusercontent.com/448627/95760718-a7dab000-0cab-11eb-8c64-15ee7cdede9d.jpg)

**Additional context**

Discord username (if different from GitHub): St54Kevin

**Comment by MisterChocker** [comment]
Flight Factor printed one on the table. I found that very handy. But I also have a physical checklist.

**Comment by St54Kevin** [comment]
Yes but its on the EFB which isnt handy because you have to look and check for every item. If it would be a popup you have both in view

**Comment by MisterChocker** [comment]
ah u got me... yeah u right that would be nice.

**Comment by Pleasure0102** [comment]
Not possible until we can change the model unfortunately.
